pkg/didcomm/protocol/introduce/service.go

```diff
@@ -268,6 +268,24 @@ func (s *Service) sendMsgEvents(msg *service.StateMsg) {
 	}
 }
 
+// Opt option for Continue function
+type Opt func(msg *metaData)
+
+// WithSkipProposal is used when the client received request message and he has a public invitation
+func WithSkipProposal() Opt {
```

Comment:
The consumer of framework shouldn't have any references to service pkg ie `pkg/didcomm/protocol/...`. They need to refer only client pkg ie, `pkg/client/...`.